ludwig/utils/config_utils.py

```diff
@@ -69,17 +68,7 @@ def merge_config_preprocessing_with_feature_specific_defaults(
 
 
 @DeveloperAPI
-def merge_fixed_preprocessing_params(
-    feature_type: str, preprocessing_params: Dict[str, Any], encoder_params: Dict[str, Any]
-) -> Dict[str, Any]:
-    """Update preprocessing parameters if encoders require fixed preprocessing parameters."""
-    encoder_type = encoder_params.get(TYPE, get_default_encoder_type(feature_type))
-    encoder_class = get_encoder_cls(feature_type, encoder_type)
-    return merge_dict(preprocessing_params, encoder_class.get_fixed_preprocessing_params(encoder_params))
-
-
-@DeveloperAPI
-def get_default_encoder_type(feature_type: str) -> str:
+def get_default_encoder_type(model_type: str, feature_type: str) -> str:
```

Comment:
Might be missing something here, but where is `model_type` being used?

Reply:
Good catch! Seems we don't actually use this function anymore, so removed.